source/computer/fedora-setup.rst

```diff
@@ -142,6 +142,19 @@ Fortran 编译器，其提供了 ``gfortran`` 命令::
 
     $ sudo dnf install gcc-gfortran
 
+Intel 软件开发工具包
+^^^^^^^^^^^^^^^^^^^^
+
+`Intel oneAPI Toolkits <https://software.intel.com/content/www/us/en/develop/tools/oneapi.html>`__
+是英特尔最新的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器。此外还有 MKL 数学库、MPI 并行库等。
+该工具包是免费的，不需要许可证。
+
+日常科研中安装 Base Toolkit 和 HPC Toolkit 即可，
+从官网\ `下载安装程序 <https://software.intel.com/content/www/us/en/develop/tools/oneapi/all-toolkits.html>`__\ 。
+下载时，选择 Local Installer，使用 ``wget`` 命令下载安装脚本。
+然后使用 ``sudo`` 运行安装脚本。工具包默认安装在 :file:`/opt/intel/oneapi` 目录下。
+安装完成后，按照指示设置好环境变量就可以使用该工具包了。
```

Comment:
why not use the `yum` or `dnf`?

https://software.intel.com/content/www/us/en/develop/articles/installing-intel-oneapi-toolkits-via-yum.html

Reply:
I did not try it. Intel recommend local installer. Will try it.

Reply:
We can use apt and dnf for Ubuntu and Fedora. For macOS, we can only use local installer.

See ed48372ad8c9fe712a96796717ee8c1d2ffbbbd8.